lib/src/delegates/assets_picker_text_delegate.dart

```diff
@@ -141,7 +141,7 @@ class EnglishTextDelegate implements AssetsPickerTextDelegate {
   String select = 'Select';
 
   @override
-  String unSupportedAssetType = 'Unsupported HEIC asset type.';
+  String unSupportedAssetType = 'Unsupported asset type.';
```

Comment:
Currently, we can recognize all types of assets if they're recognized by the OS, so I suggest we keep this value and the field without making a default value breaking change (There will be some people that might suffer from this).

Reply:
Ok, I will revert this change.

Reply:
Revert value changed for unSupportedAssetType (See last commit)